Mathlib/GroupTheory/OrderOfElement.lean

```diff
@@ -1303,6 +1303,10 @@ theorem IsOfFinOrder.prod_mk : IsOfFinOrder a → IsOfFinOrder b → IsOfFinOrde
 #align is_of_fin_order.prod_mk IsOfFinOrder.prod_mk
 #align is_of_fin_add_order.prod_mk IsOfFinAddOrder.prod_mk
 
+@[to_additive]
+lemma orderOf_mk {a : α} {b : β} : orderOf (a, b) = Nat.lcm (orderOf a) (orderOf b) := by
+  rw [← mul_one a, ← one_mul b, Prod.orderOf, mul_one, one_mul]
```

Comment:
It's fine to have this as a rewrite lemma since it's not exactly `Prod.orderOf` without beta reduction, but they are still defeq :golfing:
```suggestion
lemma orderOf_mk {a : α} {b : β} : orderOf (a, b) = Nat.lcm (orderOf a) (orderOf b) :=
  (a, b).orderOf
```

Reply:
Good point about DefEq. Thanks!